### PR TITLE
Clip global norm

### PR DESCRIPTION
**Pull request by noobOriented**
把 clip global norm 加進 GradientClipping wrapper 裡

### CODE REVIEW

**Comment by SoluMilken** [comment]
global norm 和 普通的 norm 差在哪兒？ :thinking:

**Comment by noobOriented** [comment]
普通的是每個 tensor 各自 clip 自己的 norm
global 是大家都 clip by `sum(L2(t) for t in tensors)`

**Comment by noobOriented** [comment]
如果把每個 tensor 的每個 elements 都當成一個 dimension
global norm 比較合理

https://www.tensorflow.org/api_docs/python/tf/clip_by_global_norm?version=stable
提到 `This is the correct way to perform gradient clipping (for example, see Pascanu et al., 2012 (pdf)).`

ICLR 2020: score 888
https://openreview.net/forum?id=BJgnXpVYwS

File: talos/__version__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 __title__ = 'talos'
-__version__ = '1.6.3'
+__version__ = '1.6.4'
```
Comment (SoluMilken) [inline talos/__version__.py]: 版號 要不要參考一下 大家都怎麼打
通常最小板號 是修 bug
